### ISSUE TITLE
Facebook App Events not working on Android

### ISSUE THREAD

**Issue by BerniWittmann** [segmentio/analytics-react-native#526]
When setting up the FacebookAppEventsPlugin I'm receiving an error `Got unknown argument class: String[]` on app start. This only happens on Android, on ios it seems to work fine. This error is not happening if i uncomment the line that adds the plugin.

Although unrelated I'm still getting a type error in the `segmentClient.add({ plugin: new FacebookAppEventsPlugin() });` call (this has been reported already, but seems to persist in the newest version). 

Any advice on how to resolve/debug this issue is much appreciated :) 

- `analytics-react-native` version: 2.1.12
- Integrations versions (if used): @segment/analytics-react-native-plugin-facebook-app-events 0.2.1
- React Native version: 0.63.3
- iOS or Android or both: Android (although type error is both)
- react-native-fbsdk-next version: 4.6.0

**Steps to reproduce**

add Facebook App events plugin: 

```

import { FacebookAppEventsPlugin } from '@segment/analytics-react-native-plugin-facebook-app-events';

 const segmentClient = createClient({
    debug: devEnvName() === 'dev',
    writeKey,
    trackAppLifecycleEvents: true,
    collectDeviceId: true,
  });
segmentClient.add({ plugin: new FacebookAppEventsPlugin() });
```

**Expected behavior**

App Starts without Error

**Actual behavior**

Error appears `Got unknown argument class: String[]`

**Comment by oscb** [comment]
@BerniWittmann I got a repro of this issue, seems to be coming from the underlying library the plugin uses for the FB SDK, I opened a [bug](https://github.com/thebergamo/react-native-fbsdk-next/issues/255) over there. I'll debug and see if I can work something out.

**Comment by oscb** [comment]
Just submitted a [PR](https://github.com/segmentio/sovran-react-native/pull/28) on the depedency library, hopefully it can be released soon! Else we will figure out how to release a patch on our plugin.

**Comment by oscb** [comment]
Closing as this was fixed in [v0.2.2](https://www.npmjs.com/package/@segment/analytics-react-native-plugin-facebook-app-events/v/0.2.2) of `@segment/analytics-react-native-plugin-facebook-app-events`
